transactions/update-contract/2023/feb-23/README.md

```diff
@@ -19,6 +19,88 @@ Used this to generate args:
 
 `cat "./FlowIDTableStaking.cdc" | xxd -p -c0 | tr -d '\n'`
 
+Verified using:
+```
+$ cat arguments-update-contract-FlowIDTableStaking-mainnet.json | jq '.[] | .value' | xxd -r -p > temp1.txt
+$ diff temp1.txt FlowIDTableStaking.cdc
+(Should produce no difference)
+```
+
+> Tools is unable to generate the link since the argument content is too large. We will have to do this using the old way using `flow cli`
```

Comment:
we will need to do this one the old way by using the flow cli since the tool is unable to generate the link because of the large size of the argument.